### PR TITLE
0.4.3

### PR DESCRIPTION
**Pull request by akira02**
## Related Links
Hotfix 0.4.3 add l2 mainnet settings

**Asana**:  

**Mockup**:  

## Checklist

- [ ] Pasted Asana or Mockup link.
- [ ] Tagged labels.

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -1,136 +1,140 @@
...
   // Avalanche Mainnet
-  43114: "https://api.avax.network/ext/bc/C/rpc",
+  43114: 'https://api.avax.network/ext/bc/C/rpc',
   // Avalanche Fuji Testnet
-  43113: "https://api.avax-test.network/ext/bc/C/rpc",
+  43113: 'https://api.avax-test.network/ext/bc/C/rpc',
 
   // Arbitrum Testnet
-  421613: "https://endpoints.omniatech.io/v1/arbitrum/goerli/public",
+  421613: 'https://endpoints.omniatech.io/v1/arbitrum/goerli/public',
```
Comment (mordochi) [inline src/constants.ts:26]: Do we need to add the RPC mapping for the mainnet too?

Comment (akira02) [reply src/constants.ts:26]: It's use for fallback, no needed

Comment (akira02) [reply src/constants.ts:28]: Added

Comment (mordochi) [reply src/constants.ts:28]: thanks

**Review by mordochi — APPROVED** [review]

**Pull request merged**